### PR TITLE
Merge pull request #1 from tonkeeper/main

### PR DESCRIPTION
**Pull request by Cambaa**
Cambatonkep

Please make sure you change the original ones `.yaml` fiels in directories `accounts/`, `collections/` or  `jettons/` and do not touch automatically generated .json files in the repository root.

Пожалуйста, убедитесь, что вы изменяете исходные `.yaml` файлы в директориях  `accounts/`, `collections/` или  `jettons/` и не трогаете автоматически сгенерированные файлы `.json` в корне репозитория.

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
1062

**Review by Hamaster2 — APPROVED** [review]

**Review by Jones098 — APPROVED** [review]

**Review by Hamaster2 — APPROVED** [review]

**Review by tiaraton — APPROVED** [review]

**Review by tiaraton — APPROVED** [review]

File: __pycache__/dexes.cpython-310.pyc
Comment (ghost) [inline __pycache__/dexes.cpython-310.pyc:1]: __pycache__/dexes.cpython-310.pyc

**Review by ghost — APPROVED** [review]
__pycache__/dexes.cpython-310.pyc

**Review by Hamaster2 — CHANGES_REQUESTED** [review]
_****_

**Review by Hamaster2 — COMMENTED** [review]
_*n*

File: __pycache__/dexes.cpython-310.pyc
Comment (ayanwaar) [reply __pycache__/dexes.cpython-310.pyc:1]: its me

**Review by ayanwaar — APPROVED** [review]
all

**Review by Hamaster2 — APPROVED** [review]

**Review by Hamaster2 — APPROVED** [review]

**Review by Hamaster2 — APPROVED** [review]

File: __pycache__/dexes.cpython-310.pyc
Comment (Jones098) [reply __pycache__/dexes.cpython-310.pyc:1]: Yes

**Review by ayanwaar — APPROVED** [review]

**Review by Hamaster2 — APPROVED** [review]

**Review by Hamaster2 — APPROVED** [review]

**Review by Hamaster2 — APPROVED** [review]

**Review by Hamaster2 — APPROVED** [review]

**Review by Hamaster2 — APPROVED** [review]

**Review by Hamaster2 — APPROVED** [review]

**Review by Hamaster2 — APPROVED** [review]

File: __pycache__/dexes.cpython-310.pyc
Comment (Hamaster2) [reply __pycache__/dexes.cpython-310.pyc:1]: Tonhub

**Review by Hamaster2 — APPROVED** [review]

**Review by Hamaster2 — APPROVED** [review]

**Review by Hamaster2 — APPROVED** [review]

**Review by Hamaster2 — APPROVED** [review]

**Review by tiaraton — APPROVED** [review]

File: jettons.json
Comment (ayanwaar) [inline jettons.json:1]: holla

**Review by ayanwaar — APPROVED** [review]

**Review by ayanwaar — APPROVED** [review]

**Review by Rezacb0413 — APPROVED** [review]

**Review by Rezacb0413 — APPROVED** [review]

**Review by ghost — APPROVED** [review]
# TON Assets

This is a list of popular addresses and assets. We are open to pull request but **please read manual in the end of this readme for PR**.

## Addresses

Short list. Full in [accounts.json](accounts.json)

Name           | Address
---------------|-----------
%s


## NFT collections

Name           | Address
---------------|-----------
%s


## Jettons

Name           | Address
---------------|-----------
%s


## Structure

Directories `accounts`, `collections` and `jettons` contain source data. If you want to make pull request - **please commit to this directories only**.
`accounts.json`, `jettons.json`, `collections.json` and `README.md` contains automatically generated data. Please don't commit directly to this files.

**Review by Hamaster2 — APPROVED** [review]

**Review by Rezacb0413 — APPROVED** [review]

**Review by Rezacb0413 — APPROVED** [review]

**Review by Hamaster2 — APPROVED** [review]

File: README.md
Comment (Hamaster2) [inline README.md:1]: 17f0ce0270a6910d6d464495e3913df8c7322e59

**Review by Hamaster2 — CHANGES_REQUESTED** [review]

**Review by Hamaster2 — APPROVED** [review]

**Review by Hamaster2 — APPROVED** [review]

**Review by Hamaster2 — APPROVED** [review]

**Review by Hamaster2 — APPROVED** [review]

**Review by ReadoneHussein — APPROVED** [review]

**Review by Hamaster2 — APPROVED** [review]

**Review by Hamaster2 — APPROVED** [review]

**Review by Hamaster2 — APPROVED** [review]

**Review by Hamaster2 — APPROVED** [review]

File: jettons.json
Comment (Hamaster2) [reply jettons.json:1]: [](#)

**Review by Hamaster2 — APPROVED** [review]

File: README.md
Comment (ghost) [reply README.md:1]: I want to work on this

**Comment by ghost** [comment]
how to fix this

File: __pycache__/dexes.cpython-310.pyc
Comment (high089) [reply __pycache__/dexes.cpython-310.pyc:1]: __pycache__/dexes.cpython-310.pyc

**Review by ghost — APPROVED** [review]
https://github.com/tonkeeper/ton-assets/pull/1062/commits/d80f3dfe8dc7031af0410baf9174226e17958eb1

**Comment by ghost** [comment]
https://github.com/tonkeeper/ton-assets/pull/1062/commits/d80f3dfe8dc7031af0410baf9174226e17958eb1

**Review by Hamaster2 — APPROVED** [review]

**Review by Hamaster2 — APPROVED** [review]

**Review by Hamaster2 — APPROVED** [review]

**Review by Hamaster2 — APPROVED** [review]

**Review by Hamaster2 — CHANGES_REQUESTED** [review]
# ۵۶

**Review by Hamaster2 — APPROVED** [review]

File: __pycache__/dexes.cpython-310.pyc
Comment (Hamaster2) [reply __pycache__/dexes.cpython-310.pyc:1]: <=>

**Review by Hamaster2 — APPROVED** [review]

**Review by Hamaster2 — COMMENTED** [review]
<=>

**Review by Hamaster2 — APPROVED** [review]

**Review by Hamaster2 — APPROVED** [review]

**Comment by Hamaster2** [comment]
56#

**Comment by Hamaster2** [comment]
<=> Cambatonkep
> 
> Please make sure you change the original ones `.yaml` fiels in directories `accounts/`, `collections/` or  `jettons/` and do not touch automatically generated .json files in the repository root.
> 
> 
> Пожалуйста, убедитесь, что вы изменяете исходные `.yaml` файлы в директориях  `accounts/`, `collections/` или  `jettons/` и не трогаете автоматически сгенерированные файлы `.json` в корне репозитория.

**Comment by Hamaster2** [comment]
#۵۶####

**Review by Hamaster2 — APPROVED** [review]

**Review by Hamaster2 — APPROVED** [review]

**Review by high089 — APPROVED** [review]

**Comment by ayanwaar** [comment]
093529635b9f2a94083359e9c446a73dd15dae3a

**Comment by ayanwaar** [comment]
b89dd78532d194ef24e8ecc458790d0214d8c25d

**Pull request closed without merging**